app/views/experience_stages/show.html.erb

```diff
@@ -43,7 +43,8 @@
     <div class='row'>
       <% @themes.each do |theme| %>
         <div class='col-sm-3 nav-link' href='#theme-<%= theme.id %>' du-smooth-scroll du-scrollspy offset='110'>
-          <p class='title'><%= theme.title.html_safe %></p>
+          <p class='title lg'><%= theme.title.html_safe %></p>
```

Comment:
I think that means it's visible on every class above that size. Also note that this entire menu is hidden on xs anyways, so we don't need to define functionality for that.